flake.nix

```diff
@@ -6,29 +6,25 @@
 
   outputs = inputs: inputs.flake-utils.lib.eachSystem [ "x86_64-linux" "x86_64-darwin" ] (system:
     let
-      mob-overlay = self: super: {
-        mob = super.mob.overrideAttrs (old: rec {
-          version = "4.0.0";
-
-          src = super.fetchFromGitHub {
-            owner = "remotemobprogramming";
-            repo = old.pname;
-            rev = "v${version}";
-            sha256 = "sha256-c6Feex0FGxxOWBEHxe0GqPHv65EwMMdxIeehZUGbl0Q=";
-          };
-        });
+      pkgs = import inputs.nixpkgs { system = system; overlays = [ ]; };
+      haskellPackages = pkgs.haskell.packages.ghc92.override {
+        overrides = self: super: {
+          riscv-hs = self.callCabal2nix "risv-hs" ./. { };
+        };
       };
-      pkgs = import inputs.nixpkgs { system = system; overlays = [ mob-overlay ]; };
-      haskellPackages = pkgs.haskell.packages.ghc92;
+      haskellDeps = drv: with builtins; concatLists (attrValues drv.getCabalDeps);
     in
     {
+      packages = pkgs // {
+        inherit haskellPackages;
+      };
```

Comment:
```suggestion
```

This is an artifact of us trying things out, and is not actually used (or useful) for anything.

Reply:
Good point! However, I'd suggest that we keep at least a default package pointing to `haskellPackages.riscv-hs`.

Reply:
```suggestion
      packages.default = haskellPackages.riscv-hs;
```